components/mpas-albany-landice/bld/build-namelist

```diff
@@ -55,6 +55,7 @@ OPTIONS
      -ninst_glc            NINST_GLC for this case
      -mali_dynamic         turns on dynamic ice sheet mode
                            Options are: FALSE, TRUE
+     -glc_nzoc             number of z-ocean classes [0 | 4 | 30]
```

Comment:
Just wanting to check that these are the only 3 supported values.  It sounded in our discussion like the number was flexible.

Reply:
I see, these are hard-coded in `glc_zocnclass_init_default()`

Reply:
That's correct.  I followed the implementation of land model multiple elevation classes.  In principle any number of levels (and any array of level elevations) is supported, but in practice a small set of numbers of levels are hard-coded and each has a hard-coded array of level positions.  This seems like a reasonable tradeoff between flexibility and complexity.